core/src/main/scala/cats/data/Validated.scala

```diff
@@ -1035,6 +1044,15 @@ sealed abstract private[data] class ValidatedInstances2 {
   // scalastyle:off method.length
 }
 
+private[data] class ValidatedSelective[E: Semigroup] extends ValidatedApplicative[E] with Selective[Validated[E, *]] {
+  override def select[A, B](fab: Validated[E, Either[A, B]])(ff: Validated[E, A => B]): Validated[E, B] =
```

Comment:
I'm a bit surprised this is lawful since if there is a failure in fab and ff you only see the fab failure.

Would it still pass the laws if you accumulate any errors from ff in the fab fail case?

Then again, if select is trying to be minimal here, I guess we don't want to do that since we want to avoid ff if at all possible.

Reply:
Yes, I think this could be a `RigidSelective`.  A rare one that isn't also a `Monad`.

Reply:
Is now rigid.